FireRender.Maya.Src/AlembicWrapper.hpp

```diff
@@ -0,0 +1,551 @@
+/**********************************************************************
```

Comment:
please move this file to separate directory. Would it be reasonable to move it to SharedComponents ?